projects/claystack/index.js

```diff
@@ -0,0 +1,25 @@
+const sdk = require('@defillama/sdk');
+const clayMainAbi = require('./clayABIs/clayMain.json');
+const web3 = require('../config/web3.js');
+
+const clayAddresses = {
+  clayMatic: "0x91730940DCE63a7C0501cEDfc31D9C28bcF5F905",
+};
+
+const coinAddresses = {
+  matic: "0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0", 
+};
+
+async function getClayMaticTvl(block) {
+  const clayMaticInstance = new web3.eth.Contract(clayMainAbi, clayAddresses.clayMatic);
+  const deposits = await clayMaticInstance.methods.funds().call();
+  return { [coinAddresses.matic]: deposits.currentDeposit };
+}
+
+module.exports = {
+
+  polygon: {
```

Comment:
If the staking all happens on ethereum, should this be changed to 'ethereum'?

Reply:
Staking happens on ethereum chain but we are staking Matic(or polygon) tokens. So we used polygon as notion.

Reply:
resolved export to ethereum

Reply:
For Lido, we count their MATIC staking on ethereum, because the staking action is done on ethereum and the deposit receipt token is issued on ethereum. So I think we should do the same here, since the defi activity doesnt really take place on polygon, even though your protocol sends funds to polygon internally

Reply:
Yeah, its been updated to ethereum